src/app/components/pages/Witnesses.jsx

```diff
@@ -76,6 +78,10 @@ class Witnesses extends React.Component {
             const owner = item.get('owner');
             const thread = item.get('url');
             const myVote = witness_votes ? witness_votes.has(owner) : null;
+            const signingKey = item.get('signing_key');
+            const witnessIsDisabled = /STM1111111111111111111111111111111114T1Anm/.test(signingKey);
```

Comment:
This needs to be a constant with a good name and explanation. 

Anything that _might-ever be referenced by other components_ probably should go in src/shared/constants otherwise @ the top of the file.